test/Swashbuckle.AspNetCore.Newtonsoft.Test/SchemaGenerator/NewtonsoftSchemaGeneratorTests.cs

```diff
@@ -322,6 +322,9 @@ public void GenerateSchema_SetsValidationProperties_IfComplexTypeHasValidationAt
             Assert.Equal(3, schema.Properties["StringWithLength"].MaxLength);
             Assert.Equal(1, schema.Properties["ArrayWithLength"].MinItems);
             Assert.Equal(3, schema.Properties["ArrayWithLength"].MaxItems);
+            Assert.Equal(true, schema.Properties["IntWithRange"].ExclusiveMinimum);
+            Assert.Equal(true, schema.Properties["IntWithRange"].ExclusiveMaximum);
+            Assert.Equal("byte", schema.Properties["StringWithBase64"].Format);
```

Comment:
Feels like we should be asserting that the `type` is `string` as well.

Reply:
I do not see the point, because right now if you put the Base64StringAttribute over a thing that it's not a string it will generate the format as well.

I doubt about it as you can see here: https://github.com/domaindrivendev/Swashbuckle.AspNetCore/pull/2958/commits/454168b213feb3fb1421ff9416c088dabecb4b12

But I decided to apply the same policy as RangeAttribute, if you put the RangeAttribute over a thing that is not a number it will put in the Swagger the values. For example if you put the RangeAttribute over a DateTimeOnly you will see the Swagger generated with Minimum and Maximum values.

If you think that we should apply the Base64StringAttribute over a property that is a string or the RangeAttribute over a property that is a number I think that should be a different issue

Reply:
I still think it's worth asserting on as that's the intended outcome of the schema. It might not be related to the fact it's added on _any_ type, but this test is specifically testing base64 strings so I think we should assert that fact.

Reply:
It's also one line of code and trivial to add.

Reply:
Ok, as soon as I have access to the computer I'll add it